pushapkscript/docker.d/init_worker.sh

```diff
@@ -96,6 +96,8 @@ case $COT_PRODUCT in
         test_var_set 'GOOGLE_CREDENTIALS_FOCUS'
         test_var_set 'GOOGLE_PLAY_SERVICE_ACCOUNT_REFERENCE_BROWSER'
         test_var_set 'GOOGLE_CREDENTIALS_REFERENCE_BROWSER'
+        test_var_set 'AMAZON_CLIENT_ID'
```

Comment:
I'm piggybacking on this PR some missing credentials as well to save a deployment later on.